src/sv-pipeline/04_variant_resolution/scripts/clean_vcf_part5_update_records.py

```diff
@@ -123,18 +123,22 @@ def main():
 
                 if gt5kb_del:
                     for sample_obj in record.samples.itervalues():
-                        if not sample_obj['GQ'] is None and sample_obj['RD_CN'] >= 2:
+                        if not sample_obj['GQ'] is None and \
```

Comment:
Would `sample_obj['GQ'] is not None` be better?

Reply:
Yes definitely more pythonic/readable. According to [this](https://docs.python.org/3/reference/expressions.html#operator-precedence) the order of operations is `is` > `not` > `and` so the two are equivalent though.